snippets/markdown.json

```diff
@@ -249,5 +249,32 @@
 				"   {{< nextlink href=\"link/here\" >}}Link desctiption goes here{{< /nextlink >}}",
 				"{{< /whatsnext >}}"
 			]
+	},
+	"Eng Blog Frontmatter":{
+		"prefix": ";;eng_blog_frontmatter",
+		"body": [
+			"---",
+			"authors:",
+			"- email: author@datadoghq.com",
+			"  name: author-full-name",
+			"  image: author-headshot-image.jpg",
+			"blog: engineering",
+			"date: YYYY-MM-DD",
+			"publishDate: YYYY-MM-DD",
+			"description: \"A-short-summary-of-the-post-to-be-used-on-the-front-page-of-the-blog\"",
+			"specific_interest:",
```

Comment:
Should`specific_interest` have an example value?

Reply:
So it's actually optional. I'm trying to find out which fields are required so that I can note it somehow. It might be in the next iteration though.

Reply:
I updated it so it says optional!